report/brugertests.tex

```diff
@@ -44,3 +44,29 @@ \subsection{Brugertest 2}
     \item Tilføj egne planter og sorter, samt at kunne give sådato, data om forkultivering, noter mv.
     \item Todo-liste og generelle noter
 \end{enumerate}
+
+\subsection{Brugertest 3}
+
+Ved tredje brugertest med bruger 1 efter sprint 7, fik vi opklaret nogle uklarheder i forhold til hvordan brugeren ønsker at holde styr på forspiring, samt hvilke detaljer om sine planter hun ønsker at have tilgængelige.
+
+Der blev gjort udtryk for at det var forvirrende at se alle informationer om en plante samlet på en side, så det blev ønsket opdelt i informationer om den specifikke plante og informationer om den overordnede plante generelt. Til det ønskede brugeren at hun kunne indtaste dato for hvornår hun har forspiret en plante og ikke om hvorvidt den er forspiret eller ej.
+
+Brugeren var stadigvæk forvirret over måden som et bed blev illustreret på, fordi hver række i et bed er kvadratiske. Til det foreslog hun, at et bed blev lavet mere aflangt på den ene led, så det er tydeligt at hver firkant er en række (eller en del af en række) og ikke et bed. Hun tilføjede herudover, at hun godt kunne tænke sig at der var plads til minimum fem rækker / kolonner på hver led i et bed.
+
+Vi opdagede at en af de skærmbilleder som brugeren kom til, når hun skulle oprette et nyt bed var ikke lavet til skærmstørrelsen på hendes telefon, og noget af teksten blev derfor ikke vist.
+
+Brugeren gav udtryk for at hun hellere ville have en oversigt over de planter, der er sæson for frem for at få at vide, at der kan plantes planter i hendes bed på nuværende tidspunkt. Denne oversigt blev der lavet en række forslag til hvordan den kunne blive indbygget:
```

Comment:
Her husker jeg det igen lidt anderledes - hun var forvirret og synes der manglede forklaring, så vi foreslog, at det skulle vises på en helt anden måde, hvilket hun syntes godt om. Her tilføjede hun også, at man det ikke gav så meget mening at have det på den måde, da man jo vil så de frø man har, og appen mere skal give noget information på forhånd, man kan reagere på senere. Den tankegang har vi generelt taget meget til os, og vores fremadrettede syn på appen er, at den skal fungere som visualisering af mange informationer frem for at styre din hånd